test.sh

```diff
@@ -1,5 +1,67 @@
 #!/bin/bash
 
 # main test runner for heketi
+# Executes all executable scripts under tests dir
+# in sorted order.
 
-# TODO: add tests here...
+vecho () {
+	if [ "$verbose" = "yes" ] ; then
+		echo "$*"
+	fi
+}
+
+CLI=$(getopt -o xvh --long exitfirst,verbose,help -n $0 -- "$@")
```

Comment:
make sure to quote all string var extrapolations. (like `CLI="$(getopt ....` )

Reply:
Will fix.